### DISCUSSION TITLE
Open-Source Modeling Framework in R

### DISCUSSION THREAD

**Discussion by ekuo70** [actuarialopensource/website#38] [Team Posts]
I have been working on building an open-source life insurance actuarial modeling tool using R for some time.  I have also started creating contents for documentations and sample codes in website [rmodel.io](https://rmodel.io).  There is not much content yet, but I plan to add more down the road.  Today I just added a post regarding how to implement a level term product using the framework.

Feel free to take a look, try out the codes, and give your comment if you are interested.  Perhaps it can also serve as a "reference implementation" mentioned by @fumitoh

**Comment by fumitoh** [comment]
This is great. Maybe you want to put sample assumptions enough to model a term product here? : https://github.com/actuarialopensource/standard-model/tree/main/implementations/sampledata

For discount rates, I was thinking of using https://github.com/wjwillemse/solvency2-data tool by @wjwillemse
